### ISSUE TITLE
[BUG] Input field should have live region on character count

### ISSUE THREAD

**Issue by alxzak** [kirbydesign/designsystem#4019]
[Issue]
When the user enters data into the "Name" form field, the character count (e.g., 2/30) updates dynamically, but this change is not communicated to screen readers.

[User Impact]
Screen reader users will be unaware of important updates to this content when they occur.

[Code Reference]
<kirby-form-field-message _nghost-ng-c2212097698="" ng-reflect-text="2/30">2/30
</kirby-form-field-message>

[Steps to Reproduce]

Enable a screen reader.
Using the Down Arrow key or Tab key, navigate to the dynamically updating content. Trigger it with the Enter key if necessary.
Review any announcements given by the screen reader in response to the dynamic content update.
Steps to reproduce

Enable a screen reader.
Using the Down Arrow key or Tab key, navigate to the dynamically updating content, then trigger the update.
Review any announcements given by the screen reader in response to the dynamic content update.
Issue frequency
Individual

Screen
09 Rename Profile

WCAG Success criteria
4.1.3

WCAG Conformance level
AA

[Recommendation]
Indicate live regions for dynamically changing content. Live regions can be created by adding a role attribute set to "log", "status", "alert", "progressbar", "marquee", or "timer" as appropriate. Alternatively, custom behavior can be created by using the aria-live, aria-atomic, and aria-relevant attributes. Text injected into this live region element will be announced by screen readers.

Importantly, the element with the ARIA live attributes must be available when the page loads. Otherwise, many screen readers will not detect updates to the element. Additionally, the element must be empty on page load unless an immediate screen reader announcement is desired.

[Recommended Reading]
For more information about labeling page regions, please see the MDN Web Docs website: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions

**Comment by alxzak** [comment]
<img width="1365" height="676" alt="Image" src="https://github.com/user-attachments/assets/c38a3c2b-8a99-4033-977d-94b9c8f41358" />
